models/User.js

```diff
@@ -50,9 +57,15 @@ const UserSchema = Schema({
     default: "",
   },
   reviews: {
-    type: [ReviewSchema],
+    type: [{ type: Schema.Types.ObjectId, ref: "Review" }],
```

Comment:
Does this mean that we have a separate collection for Reviews and are only storing the review id with the User?

Reply:
Yes and no, there will be a separate collection for Reviews, but Review id will exist both in User(FK) and in Review (PK). The change to this design is to achieve joining of the User collection and Review Collection